source/tests/pt_expt/model/test_export_with_comm.py

```
@@ -0,0 +1,342 @@
+        nloc=4,
+        nghost=0,
+        device=torch.device("cpu"),
+    )
+    assert len(comm_inputs) == 8
+    (
+        send_list,
+        send_proc,
+        recv_proc,
+        send_num,
+        recv_num,
+        communicator,
```

Ruff (0.15.12)

[warning] Unpacked variable `communicator` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)